lib/perfectqueue/backend/rdb_compat.rb

```diff
@@ -80,27 +80,50 @@ def initialize(client, config)
           # connection test
         }
 
-        if config[:disable_resource_limit]
+        # MySQL's CONNECTION_ID() is a 64bit unsigned integer from the
+        # server's internal thread ID counter. It is unique while the MySQL
+        # server is running.
+        # https://bugs.mysql.com/bug.php?id=19806
+        #
+        # An acquired task is marked with next_timeout and CONNECTION_ID().
+        # Therefore while alive_time is not changed and we don't restart
+        # the server in 1 second, they won't conflict.
+        if @disable_resource_limit = config[:disable_resource_limit]
+          @update_sql = <<SQL
+UPDATE `#{@table}`
+   SET timeout=?, owner=CONNECTION_ID()
+ WHERE timeout <= ? AND timeout <= ? AND created_at IS NOT NULL
```

Comment:
`timeout <= ? AND timeout <= ?` is for sharing params both mode (disable_resource_limit or not).
This PR is already not sharing params due to differ params order both mode.
Simply we can use `timeout <= ?`.